CITATION.cff

```diff
@@ -2,6 +2,6 @@ message: "If you use this software, please cite it as below."
 authors:
 - family-names: "Cook County Assessor's Office"
 title: "PTAXSIM"
-version: 0.6.1
+version: 0.6.2
```

Comment:
Bumping the package version solely due to the addition of 2023 testing data. No actual package functionality has changed.